config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -127,6 +127,59 @@ presubmits:
           --timeout=65m
           "--node-args=--image-config-file=${GOPATH}/src/k8s.io/test-infra/jobs/e2e_node/containerd/containerd-release-1.7-presubmit/image-config-presubmit.yaml -node-env=PULL_REFS=$(PULL_REFS)"
 
+  - name: pull-containerd-node-e2e-1-7-sandboxed
+    always_run: true
```

Comment:
(If you can't test this locally) Can you mark this `always_run: false` to start, and when we verify this is working properly we can set it `true`?

Reply:
Updated to set `always_run: false`.

I tried testing with `config/pj-on-kind.sh pull-containerd-node-e2e-1-7-sandboxed` per [the docs](https://docs.prow.k8s.io/docs/build-test-update/#how-to-test-a-prowjob) but encountered issues with it trying to find `GOOGLE_APPLICATION_CREDENTIALS`.